app/core/vectordb/postgres4langchain.py

```diff
@@ -116,8 +116,8 @@ def get_relevant_documents(self, query: list, **kwargs) -> List[LangchainDocumen
             cur = self.db_conn.cursor()
             cur.execute(
                 "SELECT source_id, document FROM embeddings "+\
-                "where label=%s ORDER BY embedding <=> %s LIMIT %s;",
-                (self.label, np.array(query_vector), self.query_limit))
+                "where label = ANY(%s) ORDER BY embedding <=> %s LIMIT %s;",
```

Comment:
Is this vulnerable to SQL injection?

Reply:
As per the recommendations I see [here](https://realpython.com/prevent-python-sql-injection/#passing-safe-query-parameters), it is safe. We don't use python's string concatenation or string formatting. Instead, use psycopg2's execute function's second parameter to pass the values to be substituted in the query.